### ISSUE TITLE
Validator does not work with OpenAPI v3 definition with empty servers object.

### ISSUE THREAD

**Issue by HugoMario** [cdimascio/express-openapi-validator#137]
This issue can be reproduced with the given example on repo, just switching
https://github.com/cdimascio/express-openapi-validator/blob/master/example/example.yaml#L10
```yaml
servers:
  - url: http://petstore.swagger.io/v1
```
for
```yaml
servers: []
```

endpoints should be changes on https://github.com/cdimascio/express-openapi-validator/blob/master/example/app.js#L36-L57 from 
```
'/v1/pets'
```
to 
```
'/pets'
```
other way, we would get a `404` `v1/pets` not found.

So having that, requesting to example server using the given CURL samples on page https://github.com/cdimascio/express-openapi-validator/tree/master/example#validate-a-query-parameter-with-a-value-constraint
will not work and endpoints will be reached.

The reason of this behavior is here:
https://github.com/cdimascio/express-openapi-validator/blob/master/src/framework/util.ts#L54

So, changing
```js
if (!servers) {
    return [new BasePath({ url: '' })];
 }
```
to
```js
if (!servers || servers.length == 0) {
    return [new BasePath({ url: '' })];
 }
```
will solve the issue

**Comment by cdimascio** [comment]
Thanks @HugoMario. I will get this change in sometime today. Feel free to submit a PR if you like and I'll gladly merge it. Thanks again!

**Comment by HugoMario** [comment]
sure, i'm on it

**Comment by cdimascio** [comment]
you fix is available in `v2.16.1` thanks again
